lockfile/src/python.rs

```diff
@@ -34,11 +32,19 @@ impl Parse for PyRequirements {
     }
 
     fn is_path_lockfile(&self, path: &Path) -> bool {
-        path.file_name() == Some(OsStr::new("requirements.txt"))
+        let file_name = match path.file_name().and_then(|f| f.to_str()) {
+            Some(f) => f,
+            None => return false,
+        };
+        file_name.starts_with("requirements") && file_name.ends_with(".txt")
     }
 
     fn is_path_manifest(&self, path: &Path) -> bool {
-        path.file_name() == Some(OsStr::new("requirements.txt"))
+        path.file_name() == Some(OsStr::new("requirements.in"))
+            || path.file_name() == Some(OsStr::new("pyproject.toml"))
+            || path.file_name() == Some(OsStr::new("setup.cfg"))
+            || path.file_name() == Some(OsStr::new("setup.py"))
+            || self.is_path_lockfile(path)
```

Comment:
I feel like this might get dangerous when someone comes up with a `lockfile.txt` in the future that isn't supposed to be a manifest at all?

Maybe it's better to be safe here and just move the logic to a `fn is_requirements_txt` method or something which is then called by both?